src/main/java/seedu/trackermon/ui/CommandBuilder.java

```diff
@@ -2,6 +2,9 @@
 
 import javafx.beans.property.SimpleStringProperty;
 
+/**
+ * The UI component that is responsible for creating the command information to be displayed in {@code Tableview}.
```

Comment:
Change to `Creates the command information to be displayed in {@code Tableview}.`?

Reply:
Isn't this a class header, the verb rule only applies to methods header?